ios-continuous-integration_5c8a397c2c7d3a154460db00.md

```diff
@@ -123,228 +122,218 @@ blocks:
+  - cache store
 ```
 
-In example configuration, the first command restores dependencies from cache
-using partial cache key matching:
+In the example configuration, the first command restores dependencies from
+cache using partial cache key matching:
 
-1. Ideal scenario is to match `gems-$SEMAPHORE_GIT_BRANCH-$(checksum Gemfile.lock)`.
+1. The ideal scenario is to match `gems-$SEMAPHORE_GIT_BRANCH-$(checksum Gemfile.lock)`.
    This means that we've found dependencies from the current Git branch, and
    current content of dependencies' lock file. `$SEMAPHORE_GIT_BRANCH` is one of
    the [environment variables][env-vars] provided by Semaphore in every job.
    [`checksum`][checksum] is a script that returns MD5 sum of provided file.
 2. The second best option is to find any previous version of dependencies for
    the current Git branch, using `gems-$SEMAPHORE_GIT_BRANCH-` as key.
-3. Last option is to reuse the last available cache from the master branch.
+3. The last option is to reuse the last available cache from the master branch.
 
-The `cache restore` command will not fail in case of a cache miss. In case of a
-cache hit, `bundle install` will complete in about a second.
+The `cache restore` command will not fail in case of a cache miss. In the case
+of a cache hit, `bundle install` will complete in about a second.
 
-After installing the dependencies, we store them in cache for reuse in upcoming
-blocks and future workflows.
+After installing the dependencies, Semaphore stores them in cache for reuse in
+upcoming blocks and future workflows.
 
-### Selecting Xcode version
+### Selecting an Xcode version
 
-You can find the list of currently available versions of Xcode in the [macOS
-Mojave image reference][macos-mojave].
+You can find the list of available versions of Xcode on the
+[Xcode 10](macos-mojave-xcode10) or [Xcode 11](macos-mojave-xcode10) image
+references. Select the desired version for your project with `xcversion`.
 
 ``` yaml
-        commands:
-          - bundle exec xcversion select 10.2
+commands:
+  - bundle exec xcversion select 11.1
 ```
 
 ### Running tests
 
-Semaphore can run tests of iOS and Mac apps on a simulator or connected device.
-If you're using Fastlane, see documentation regarding [fastlane
-scan][fastlane-scan].
+Semaphore can run tests of apps built with Xcode either on a simulator or a
+connected device. If you're using fastlane, see the
+[fastlane scan][fastlane-scan] documentation.
+.
 
 ``` yaml
-        commands:
-          - bundle exec fastlane test
+commands:
+  - bundle exec fastlane test
 ```
 
 ### Building your app
 
-We build our app by running `fastlane build`. [The Semaphore Fastlane
-plugin][fastlane-plugin] ensures that by default the process runs smoothly with
-a temporary keychain.
+The example project is set to build the app with `fastlane build`.
+The [Semaphore fastlane plugin][fastlane-plugin] ensures that by default the
+process runs smoothly with a temporary keychain.
```

Comment:
We should probably add that the keychain is automatically created by the plugin (as mentioned in the plugin's Readme).